app/services/clients/connector.client.js

```diff
@@ -257,6 +257,21 @@ ConnectorClient.prototype = {
     return response.data
   },
 
+  /**
+   * Retrieves the accepted card Types for the given account
```

Comment:
service external id and account type is a more appropriate description to help differentiate from old gateway account based endpoints